### ISSUE TITLE
setWireTimeoutUs() referenced twice in documentation for setWireTimeout()

### ISSUE THREAD

**Issue by paynterf** [arduino/ArduinoCore-avr#353]
The documentation for setWireTimeou() references setWireTimeoutUs() instead.

**Comment by matthijskooijman** [comment]
Do you mean the comments in the source code? Or is there some reference documentation online already?

That function was renamed just before merging #107, probably the documentation wasn't updated. Should be fairly trivial to fix.

**Comment by paynterf** [comment]
Yep - just the comments in the Wire.cpp source code, directly above the function itself
